http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoException.scala

```diff
@@ -13,10 +13,17 @@ import scala.util.control.NoStackTrace
 class LeoException(val message: String = null,
                    val statusCode: StatusCode = StatusCodes.InternalServerError,
                    val cause: Throwable = null,
+                   extraMessageInLogging: String = "",
                    traceId: Option[TraceId]
 ) extends WorkbenchException(message, cause) {
   override def getMessage: String = if (message != null) message else super.getMessage
 
+  def getLoggingMessage: String = s"${getMessage}. ${extraMessageInLogging}"
+  def getLoggingContext: Map[String, String] = traceId match {
+    case Some(value) => Map("traceId" -> value.asString)
+    case None        => Map.empty
+  }
+
```

Comment:
`getMessage` is user facing. Adding `getLoggingMessage` so that we can add more debug info in logs that's a bit more sensitive to send to users